webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -701,17 +701,17 @@ <h2>Crashing Thread ({{ crashing_thread }}){% if parsed_dump.threads[crashing_th
                     </thead>
                     <tbody>
                       {% for frame in parsed_dump.threads[crashing_thread].frames %}
-                        <tr class="{% if frame.missing_symbols %}missingsymbols{% endif %}">
-                          {% if frame.truncated %}
-                            <td colspan="5">truncated {{ frame.truncated|digitgroupseparator }} frames...</td>
-                          {% else %}
-                            <td>
+                        {% if frame.truncated %}
+                          <tr><td colspan="5">truncated {{ frame.truncated|digitgroupseparator }} frames...</td></tr>
```

Comment:
We'll never have a case where the truncated indicator has missing symbols, so we can do a minimal row.